tools/nix/README.md

```diff
@@ -0,0 +1,77 @@
+# Nix
+
+This prototype [Nix Flake](https://zero-to-nix.com/concepts/flakes) offers a
+self-contained, versioned, reproducible development environment for working on
+the Elixir/Rust libraries offered within this open-source repository. This flake
+is highly experimental with no stability or suitability guarantees, and not
+encouraged for use by external contributors.
```

Comment:
> This flake is highly experimental with no stability or suitability guarantees, and not encouraged for use by external contributors.

We can remove this.

Reply:
I recommend to remove this later when we have user interest and can prioritize efforts around maintainability of it.